contrib/devtools/symbol-check.py

```diff
@@ -151,12 +151,12 @@ def read_symbols(executable, imports=True) -> List[Tuple[str, str, str]]:
         raise IOError('Could not read symbols for {}: {}'.format(executable, stderr.strip()))
     syms = []
     for line in stdout.splitlines():
-        line = line.split()
-        if 'Machine:' in line:
-            arch = line[-1]
-        if len(line)>7 and re.match('[0-9]+:$', line[0]):
-            (sym, _, version) = line[7].partition('@')
-            is_import = line[6] == 'UND'
+        split_line: List[str] = line.split()
```

Comment:
note: this will be removed with #20434.

Reply:
yes, this entire function doesn't exist anymore :smile: